source/blog/2020-09-25-release-notes.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: September 2020 Release Notes - Version 20.9.0
```

Comment:
What, no emoji?

Reply:
forgot the ctrl + cmd + space shortcut. Emojified 🏀